packages/web/src/pages/sign-up-page/pages/CreatePasswordPage.tsx

```diff
@@ -42,9 +48,20 @@ export const CreatePasswordPage = () => {
     (values: CreatePasswordValues) => {
       const { password } = values
       dispatch(setValueField('password', password))
+
+      if (isGuest) {
+        dispatch(
```

Comment:
just a note, that we will probably just do normal flow and call this stuff on "finish profile" page.

Reply:
yup will follow up with that